documentation/website/documentation/models.md

```diff
@@ -622,6 +622,117 @@ For backward compatibility, we allow these to be mixed with normal ports in a li
 ]
 ```
 
+#### Annotation Features
+
+In model generators we can also use annotation features, which translate to regular user features based on annotation parameter values.
+
+Config example:
+```json
+{
+{
+  "model_generators": [
+    {
+      "find": "methods",
+      "where": [
+        {
+          "constraint": "signature_match",
+          "parent": "Lcom/facebook/marianatrench/integrationtests/AnnotationFeature;",
+          "name": "getSourceWithMethodAnnotation"
+        }
+      ],
+      "model": {
+        "generations": [
+          {
+            "kind": "Source",
+            "port": "Return",
+            "via_annotation": [
+              {
+                "type": "Lcom/facebook/marianatrench/integrationtests/Path;",
+                "port": "Return"
```

Comment:
These examples need a bit of updating. This is called `"target"` by the end of the stack? May be also mention that we support these in "for_all_parameters" as well.